### PR TITLE
feat(web): Overall design refinements

### PR DESCRIPTION
**Pull request by azroberts8**
This PR adds subtle design changes to the web interface that better emulates the design of Google Photos and provides a more refined user experience by unifying some colors and tones between light and dark mode. See commit messages for complete change log.

| Before | After |
| - | - |
| Before-Light | After-Light |
| Before-Dark | After-Dark |

### CODE REVIEW

**Comment by waclaw66** [comment]
With all respect to your effort, I don't think there is need to emulate Google Photos, rather inspire with it. One of much better things that Immich has is its font, why to change it then?

**Comment by martabal** [comment]
I agree with @waclaw66. You can use the custom css option for that.

**Comment by azroberts8** [comment]
@waclaw66 I think that "inspire" is a better term than "emulate" for the changes made in this pull request. I do believe that the signature font used in the logo of Immich is one of the traits that makes Immich unique. However, in my opinion, the "Overpass" font used throughout the design of the webapp does somewhat detract from the design feeling a bit clunky compared to "Roboto".

Overall, the theme of this pull request was to make the webapp design feel a bit more refined and professional which is reflected in some of the changes such as fixing the colors of different elements such as search bars that were using close but different tones of gray, tooltips that did not stand out against the content they were over, and condensing the spacing of list items.

File: web/src/lib/components/shared-components/side-bar/side-bar-section.svelte [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <section
   id="sidebar"
   tabindex="-1"
-  class="immich-scrollbar group relative z-10 flex w-18 flex-col gap-1 overflow-y-auto bg-immich-bg pt-8 transition-all duration-200 dark:bg-immich-dark-bg hover:sm:w-64 hover:sm:border-r hover:sm:pr-6 hover:sm:shadow-2xl hover:sm:dark:border-r-immich-dark-gray md:w-64 md:pr-6 hover:md:border-none hover:md:shadow-none"
+  class="immich-scrollbar group relative z-10 flex w-18 flex-col gap-0 overflow-y-auto bg-immich-bg pt-8 transition-all duration-200 dark:bg-immich-dark-bg hover:sm:w-64 hover:sm:border-r hover:sm:pr-6 hover:sm:shadow-2xl hover:sm:dark:border-r-immich-dark-gray md:w-64 md:pr-6 hover:md:border-none hover:md:shadow-none"
```
Comment (martabal) [inline web/src/lib/components/shared-components/side-bar/side-bar-section.svelte:7]: You can remove it

**Comment by alextran1502** [comment]
Hello, thank you for the PR. I will have to reject all other changes except the change to the background of the search bars. 

My initial feelings when looking at the new design are:
- It is pretty cluttered and condensed
- We are leaning toward MD3 nowadays to be similar to the mobile app, which has more rounded corner elements
- I'm not too fond of Roboto. It has a corporation feeling and is not elegant in my eyes

**Comment by morki** [comment]
For me personally, the Immich current font is really badly readable, making me stare at every line of text 5x longer to really read it "letter by letter" :(

**Comment by jrasm91** [comment]
> For me personally, the Immich current font is really badly readable, making me stare at every line of text 5x longer to really read it "letter by letter" :(

Feel free to use the "Custom CSS" feature in the Admin > Theme Settings > Custom CSS".

**Comment by jrasm91** [comment]
Are there any plans to continue working on this and implement the requested changes? If not, I'll just close the PR.

**Pull request closed without merging**